samples/ASP.Net Metrics/MetricsShim/MetricsShim.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Newtonsoft.Json.Linq;
+
+namespace hellocs.MetricsShim
+{
+    public class MetricShim
+    {
+        public MetricExporter MetricsExp;
+        private string _json;
+        public MetricShim(string jsonString)

```

Comment:
perhaps a better solution would be use ConfigurationProvider from asp.net core itself, instead of doing parsing of json file. That'll allow config not just from appsettings.json, but from other providers as well (like env variable etc).

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.0